WebDriverAgentLib/Utilities/FBKeyboard.m

```diff
@@ -28,10 +29,14 @@ + (BOOL)typeText:(NSString *)text error:(NSError **)error
   if (![FBKeyboard waitUntilVisibleWithError:error]) {
     return NO;
   }
+
+  NSUInteger freq = [FBConfiguration maxTypingFrequency];
```

Comment:
as far as I know variable name shortenings are not very appreciated in this project ;)